src/Products/PloneKeywordManager/setuphandlers.py

```diff
@@ -23,18 +22,15 @@ def importKeywords(context):
     if len(keywordlist) < 1:
         return
 
-    site = context.getSite()
-    id = "keywords"
-    doc = getattr(site, id, None)
+    portal = context.getportal()
```

Comment:
Can you talk about getSite() vs getportal()  ?

This is a learning opportunity.

Reply:
Well, since we are in an add-on i prefer to use plone.api if possible.